.circleci/config.yml

```
@@ -1,55 +1,24 @@
 version: 2.1
 
 orbs:
-  redmine-plugin: agileware-jp/redmine-plugin@2.3.0
-  plugin-test:
-    commands:
-      run-tests:
-        parameters:
-          plugin:
-            default: ''
-            description: Plugin name. If unspecified it will be the current repository's plugin.
-            type: string
-          redmine_root:
-            default: redmine
-            description: Directory where redmine is installed
-            type: string
-        steps:
-          - run:
-              name: Setup Database
-              command: 'RAILS_ENV=test bundle exec rake db:migrate redmine:plugins:migrate'
-              working_directory: << parameters.redmine_root >>
-          - run:
-              name: Set tested plugin
-              command: |
-                PLUGIN_NAME='<< parameters.plugin >>'
-                if [ -z $PLUGIN_NAME ]; then
-                  PLUGIN_NAME=$CIRCLE_PROJECT_REPONAME
-                fi
+  redmine-plugin: agileware-jp/redmine-plugin@dev:42732ae # will change 3.1.1
```

After https://github.com/agileware-jp/redmine-plugin-orb/pull/59 is released, apply following changes:

```suggestion
  redmine-plugin: agileware-jp/redmine-plugin@3.1.1
```